previous-doc/v5.0/ref/architecture.html

```diff
@@ -0,0 +1,156 @@
+---
+layout:
+---
+<!doctype html>
+<html><head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv="content-type" content="text/html;charset=utf-8" />
+    <link rel="canonical" href="http://nhibernate.info/doc/nhibernate-reference/architecture.html" />
```

Comment:
Canonical links have been added in all archived files which were not having already one.